docs/dev/setup.rst

```diff
@@ -66,15 +73,37 @@ You can achieve this e.g. by using a ``my.cnf`` file in the location ``/etc``.
 Make sure the configuration file is used by MySQL when you start the server.
 You can find more information on `<https://dev.mysql.com/doc/refman/8.0/en/option-files.html>`__
 
+MySQL 8 Docker Container for ARM64 processors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. TODO: check if this is still needed or if the other images already run with ARM64 processors
```

Comment:
Does anyone know this by heart? Or someone who can test this easily?

Reply:
Tested this on a RPi3 and `mysql:8.0.27` is still not supported for arm64, but `mysql:8.0.29` and upwards seem to be supported.
@Mtze How do we keep the docker and non-docker versions of mysql and co. in sync? Should this be in sync to production at TUM and therefore some other repo? Depending on that I would create an issue to upgrade at least the mysql version in a follow-up.

Reply:
It seems like we are now using `mysql:8.0.30` :

- #5662

Therefore, I will delete this as it's not necessary anymore.